### PR TITLE
SAK-48362 roster. Add profile connections buttons back

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-48362

### CODE REVIEW

**Comment by ottenhoff** [comment]
Wow this is a huge PR. My main question: is all of this useful going forward for other Sakai webcomponents? Or is it really just for this one feature? 

Because I was hoping we could have a conversation about how useful/how used these social features are.

**Comment by adrianfish** [comment]
It's useful for all the components, the bit where I style up button in the shadow dom. I've done it with our css vars so it's reasonably DRY. I can split these bits out if you think that makes sense. I don't know how useful the social stuff is, tbh.

**Comment by ottenhoff** [comment]
Ok great. I can look at the social stuff later and pull real data on how often they are used.

**Comment by adrianfish** [comment]
Screenshot 2024-04-19 at 14 44 19
Screenshot 2024-04-19 at 14 43 56

File: library/src/skins/default/src/sass/modules/button/_base.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    --sui-btn-border-color: var(--button-border-color);
+    --sui-btn-hover-border-color: var(--button-hover-border-color);
+    --sui-btn-padding: #{$btn-padding-y} #{$btn-padding-x};
+    --sui-btn-box-shadow: var(--button-shadow);
+    --sui-btn-hover-box-shadow: var(--button-hover-shadow);
+    --sui-btn-font-family: #{$btn-font-family};
+    --sui-btn-font-size: #{$btn-font-size};
+    --sui-btn-line-height: #{$btn-line-height};
+    --sui-btn-font-weight: #{$btn-font-weight};
+}
```
Comment (ottenhoff) [inline library/src/skins/default/src/sass/modules/button/_base.scss:16]: does this work in dark mode?

Comment (adrianfish) [reply library/src/skins/default/src/sass/modules/button/_base.scss:16]: Good question. It should work for our vars, of course. But, the bootstrap ones. I'll test that.

**Comment by adrianfish** [comment]
Screenshot 2024-04-19 at 16 29 09

**Comment by adrianfish** [comment]
Yes, the vars work in dark mode.

**Review by ottenhoff — APPROVED** [review]

**Pull request merged**